tests/runtime/other

```diff
@@ -280,11 +280,21 @@ EXPECT stdin:1:37-41: WARNING: Can't lookup map element because it does not exis
                                            ~~~~
 TIMEOUT 1
 
-NAME per_cpu_map_if_lt
+NAME per_cpu_map_count_if
 PROG i:ms:1 { @ = count(); if (@ > 5) { printf("done\n"); exit(); }}
 EXPECT done
 TIMEOUT 5
 
+NAME per_cpu_map_count_min_if
```

Comment:
Nit, shouldn't be?
```suggestion
NAME per_cpu_map_min_if
```
There's no `count()` used.

Reply:
whoops. good catch. my brain stopped seeing the word "count"